src/ListGuesser.js

```diff
@@ -33,18 +33,21 @@ class ListGuesserComponent extends React.Component {
 
   render() {
     const {resourceSchema, fields, ...props} = this.props;
-    const children = Children.toArray(props.children);
+
+    if (!props.children) {
```

Comment:
Any reason to not destructure children, like in other guessers ?

Reply:
Because it's not a functional component. Deconstructing to a `const` is useless cause we have to override its value.